webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const webpack = require('webpack');
+require('dotenv').config();
+
+const SRC_DIR = path.join(__dirname, '/client/src');
+const DIST_DIR = path.join(__dirname, '/client/dist');
+
+module.exports = {
+  mode: process.env.NODE_ENV,
+  entry: ['webpack-hot-middleware/client', `${SRC_DIR}/index.jsx`],
```

Comment:
you have hotmiddleware here as an entry but its not used in index.js for the server

Reply:
Thanks for catching this! Included hot middleware in server file.